public/app/plugins/panel/timeseries/plugins/ContextMenuPlugin.tsx

```diff
@@ -106,7 +98,7 @@ export const ContextMenuView: React.FC<ContextMenuProps> = ({
     return null;
   }
 
-  const items = defaultItems ? [...defaultItems] : [];
+  const items: MenuItemsGroup[] = [];
```

Comment:
Isn't this a breaking change since now there won't be default items?

Reply:
hmm...would take a closer look at this. 👍